.travis.yml

```diff
@@ -10,7 +10,11 @@ go:
     - tip
 
 env:
+  global:
     - COVERALLS_TOKEN=mwTn1pOFqEOUT13vylZNHq53NanoMznO7
+    - secure: oNMBjCUdFIrMoiiea5eqNSXb46alRZACHrwuyGArje/k8vACN14nREONG0slFDCrdDPtWZxu4OVlJaujirPRnbtj3e50myE90pHqnAjYWhKhVzyg0/NK5zqcc+RrIUDs1CA0NoH1uDQRK0+yxpnfNWSGGq3KQEFmZLzPQgtJ5ikteHIf9cjAq6Su/zz4O7X7b6UJSTekHQc0UmtzCWfbLHVZLnHFjEDX2eR2pIm27b6GGOySTbw3BT8KqEfziy8275z4KBnVqS2qt+AQWtRwdvej7jiVkjZpn78OKv2hpCphnX0RBsv7OogqHivu4JBfHKh7FywwdxBLY21rIX/wEs+t+OEtazI5jhYjk7apxBzoshvfkxnVtPWUiJ3L5GQUE4YIE5DJYrWRVmCouUC8vFqbQQrXfg4FBWQ3JxfwWZXTFo6pitZV+po0k7ob+asm/n7t5UTgznQE2IjEyWzGMsDauQ9asUVN6vnaNH/sYPHVh52AR9A2mpMjatCVdjq/sQnLd/ag1lFkFRDn3yFcAisMCPBOppc81DIRStkJTHlcGT3e5fTmLVPNAHu5cBE38BXz2eqV/vy5oj/jHtHmipr93K9T2TqdsUhYBPiQKIkYqy+1DezLm5rHLAz4fPJb2InyR6EYW1nql21sOW0vampExY73qvcvlb+MAUdPSbQ=
+    - secure: QLks228wsIZw1S/LSn6arcUnyVD+OqTw6IDWNxdbtVUZ7MTIUsq3qXVSKLN+LY7T9mP02qkGfZNNyGN5vcfPSkXeCuOnnnFjIhhjHjpOvSrCcjPZtJOXmu1orYApV9eSwL2l7qnWf5s9Rfpi1OjL0ElZcQ6/t0tL0aY7KZsrBFTxB4SMP/BkXreqP73bRvhoQfidc6/u9PR1I/Ngug3tEVNKST6ykT0hrn49EKxPyCIweVgwkXm1Wji1ZzKut5iB40nw+3I0fiPDP+Sevmqz8CIMDwJZDgoPGFgJon+cY+hB3kviahhGK6h3Jg75YA61qWmVfVOBQT6lhhl2TD1haPzYYntkv05q76VY0RqBwbgscEEWEc6mvlChdt0MgeReP6RBOhh+eDUMxykvGAFc4KNa20/cTL4QrignAfdu2OoGobM9Xi1C+W/vXirPWNreMjBT9Wba1Fxq/cHHVczqKEvy8fg4Xo0RDjBVUuEmpg/LOA6A61MXxzIOGINU7DmW+XVErvB4AGHlFPreI1UsYXriZzljhRPuGkeXfCK9/JWxwjAZKGQfKHV2Rp8G1uQVJ2XgHx4YWKzu4/gd1u4XItcFqtahJ7x/DUit0c7e28yOp7dy3c1WMT3+Y7ZXHTgBnNjeYQHXzhoHIj/7mdn/1bxAATmBVVNO3DP2TOua4zg=
```

Comment:
Of course travis would recommend it.  We should still be skeptical, cautious.

The statement by @gorozco1 that "the credentials can only be decrypted by the author and travis repo" is not true.  They are decrypted by travis during any CI run on the repo for which they were encrypted, given any/all unit test code access to the variable once it is in the environment.  There's nothing preventing malicious code in tests from stealing/exposing this sensitive data.